source/globalCommands.py

```diff
@@ -569,6 +569,24 @@ def script_previousSynthSetting(self, gesture):
 		previousSettingValue = globalVars.settingsRing.currentSettingValue
 		ui.message("%s %s" % (previousSettingName, previousSettingValue))
 
+	@script(
+		# Translators: Input help mode message for toggling keyboard layout.
+		description=_("Toggles keyboard layout (desktop or laptop)"),
```

Comment:
Rather than parentheses, could we word it like
```suggestion
		description=_("Toggles keyboard layout between desktop and laptop"),
```
or
```suggestion
		description=_("Toggles between desktop and laptop keyboard layout"),
```